src/pages/tutorials/smart-rollups/index.md

```diff
@@ -1,537 +1,235 @@
+These calls are how rollups send messages back to Tezos.
 
-**Smart rollups** are an elegant solution for **horizontally scaling** the Tezos blockchain, which involves distributing the workload of the main layer (Layer 1) to external layers that perform their tasks "off-chain". In comparison, **vertical scaling** focuses on optimizing the main layer itself but is less scalable than the former.
+Smart rollups can run any kind of applications that they want, such as:
 
-Let us use an analogy: think of a company with N employees who receive increasing amounts of work every day. Eventually, the team becomes overwhelmed. While hiring more people for the team is an option, it becomes challenging for them to coordinate, which ultimately reduces their productivity.
+- Financial applications that use information and transactions from Tezos
+- Gaming applications that manipulate assets and keep them in sync with Tezos
+- Applications that run complex logic on NFTs or other types of tokens
+- Applications that communicate with other blockchains
 
-**Vertical scaling** in this scenario means providing better working equipment to increase productivity. On the other hand, **horizontal scaling** involves creating external teams that work on specific portions of the workload, reducing the need for extensive interaction with the initial team. The latter option is more scalable because one can continuously improve the equipment up to a certain point, while creating external teams can happen at any time and will always be beneficial. In our case, these external teams are the **smart rollups**.
+{% comment %}
+TODO Should this intro discuss the reveal data channel?
+{% /comment %}
 
-## 2. The Kernel
+Rollups maintain consensus by publishing the hash of their state to Tezos, which other nodes can use to verify the rollup's behavior.
+The specific way that rollups publish their states and maintain consensus is beyond the scope of this tutorial.
+For more information about rollups and their consensus mechanism, see [Smart Optimistic Rollups](../../advanced-topics/smart-rollups/).
 
-### 2.1. Definition
+This diagram shows a smart rollup interacting with layer 1 by receiving a message, running processing based on that message, and sending a transaction to layer 1:
 
-The core component of any smart rollup is the **kernel**. A kernel is a 32-bit `WebAssembly` (`WASM`) program responsible for managing input messages, updating the state of the rollup, and determining when to output messages to Layer 1. To continue with the analogy, the kernel represents the work ethic of the "external team".
+{% html htmlWrapperTag="div" %}
 
-### 2.2. `Rust`
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/74fd884e-9c71-409e-b7d3-c3a871a17178" id="KYtBl2woQVuV"></iframe></div>
```

Comment:
This diagram, is nice, but a bit misleading? It implies that the outbox message gets executed 'the block after' the original inbox message.

It also slightly weirdly separates 'run application' out of the kernel? Is this meant in a 'VM kernel' kind of way? Really what 'run application' might be is more of 'update storage' potentially?

Reply:
- Removed numbers for blocks
- Changed it to Receive message -> Process message -> Create transaction all within the kernel
- Removed numbers from the transactions and messages to avoid implying that the blocks are sequential

New diagram:
![Smart rollup tutorial](https://github.com/trilitech/tezos-developer-docs/assets/6494785/7a803a6e-e04a-496f-9629-37cecee4cdce)

Reply:
I think this makes sense! The one thing that you could do, is add a `...` between the blocks (not sure if that would be confusing or not) to properly imply they aren't immediately adjacent

Reply:
Added an ellipsis between the blocks.